stripe/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -767,6 +767,13 @@ class CardInputWidget @JvmOverloads constructor(
             }
         }
 
+        postalCodeEditText.internalFocusChangeListeners.add { _, hasFocus ->
+            if (hasFocus) {
+                scrollEnd()
+                cardInputListener?.onFocusChange(CardInputListener.FocusField.PostalCode)
+            }
+        }
```

Comment:
Does this need to be applied to `CardMultilineWidget` as well?

Reply:
good news is that it is covered there already.